google/cloud/storage/internal/grpc/stub.cc

```diff
@@ -231,9 +232,11 @@ Options DefaultOptionsGrpc(Options options) {
     options.set<UnifiedCredentialsOption>(MakeInsecureCredentials());
   }
 
+  auto const default_endpoint = google::cloud::internal::UniverseDomainEndpoint(
+      "storage.googleapis.com.", options);
   options = google::cloud::internal::MergeOptions(
       std::move(options), Options{}
-                              .set<EndpointOption>("storage.googleapis.com")
+                              .set<EndpointOption>(std::move(default_endpoint))
```

Comment:
The `move` looks like it will be a no-op given that `default_endpoint` is `const`.

Reply:
D'oh fixed. And I messed this up in `pubsub`: #13347